ocs_ci/helpers/managed_services.py

```diff
@@ -233,3 +233,24 @@ def verify_osd_used_capacity_greater_than_expected(expected_used_capacity):
             )
             return True
     return False
+
+
+def verify_osds_are_on_correct_machinepool():
```

Comment:
It would be better to not use helpers directory and store all ocs code in ocs dir (ocs/managedservice.py).